### PR TITLE
[Improvement]: Rename "Master" to "Main" or similar in all translations and in code

### LINKED ISSUE #14286: [Improvement]: Rename "Master" to "Main" or similar in all translations and in code
### Improvement description

Probably there are more terms that need to be considered.

### PR DESCRIPTION
**Pull request by robertSt7**
resolves #14286

### CODE REVIEW

File: bundles/AdminBundle/templates/admin/login/layout.html.twig [hunk 1]
```diff
@@ -70,7 +70,7 @@
                 <h2>News</h2>
                 <hr>
                 <p>
-                    <a href="#">Where is Master Data Management Heading in the Future?</a>
+                    <a href="#">Where is Main Data Management Heading in the Future?</a>
```
Comment (mcop1) [inline bundles/AdminBundle/templates/admin/login/layout.html.twig]:
```suggestion
                    <a href="#">Where is Master Data Management Heading in the Future?</a>
```

File: bundles/CoreBundle/src/Migrations/Version20230202152342.php [hunk 2]
```diff
@@ -38,7 +38,7 @@ public function up(Schema $schema): void
         $settings = SettingsStore::get(ReportConfigWriter::REPORT_SETTING_ID, ReportConfigWriter::REPORT_SETTING_SCOPE);
         $data = json_decode($settings->getData(), true);
 
-        $marketingScopes = ['analytics', 'webmastertools', 'tagmanager'];
+        $marketingScopes = ['analytics', 'webadmintoolss', 'tagmanager'];
```
Comment (mcop1) [inline bundles/CoreBundle/src/Migrations/Version20230202152342.php]:
```suggestion
        $marketingScopes = ['analytics', 'webadmintools', 'tagmanager'];
```

File: bundles/CoreBundle/src/Migrations/Version20230202152342.php [hunk 3]
```diff
@@ -63,7 +63,7 @@ public function down(Schema $schema): void
...
```
Comment (mcop1) [inline bundles/CoreBundle/src/Migrations/Version20230202152342.php]:
```suggestion
        $marketingScopes = ['analytics', 'webadmintools', 'tagmanager'];
```

Comment (brusch) [reply bundles/CoreBundle/src/Migrations/Version20230202152342.php]: It was renamed to Google Search Console anyway 😉

**Pull request merged**